### PR TITLE
CSP-54-outer-TrainingVenueDetails

### PR DESCRIPTION
**Pull request by raviudari01**

### CODE REVIEW

File: src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.Api/Controllers/ProviderLocationsGetController.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
+﻿using MediatR;
```
Comment (MarkFCain) [inline src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.Api/Controllers/ProviderLocationsGetController.cs:1]: The naming of this controller seems odd.  Wouldn't GetProviderLocationsController be better (and I know it was previously called that, I don't understand the name change)

File: src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.Api.UnitTests/Controllers/ProviderLocationsControllerTests/GetProviderLocationTests.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.RoatpCourseManagement.Api.UnitTests.Controllers.ProviderLocationsControllerTests
+{
+    [TestFixture]
+    public class GetProviderLocationTests
+    {
+        [Test, MoqAutoData]
+        public async Task GetAllLocations_ValidRequest_ReturnsLocations(
```
Comment (MarkFCain) [inline src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.Api.UnitTests/Controllers/ProviderLocationsControllerTests/GetProviderLocationTests.cs:20]: rename test, this is testing for GetProviderLocation

File: src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.Api.UnitTests/Controllers/ProviderLocationsControllerTests/GetProviderLocationTests.cs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+            var response = await sut.GetProviderLocation(ukprn,id);
+
+            var okResult = response as OkObjectResult;
+            okResult.Should().NotBeNull();
+            okResult.Value.Should().BeEquivalentTo(result.ProviderLocation);
+        }
+
+        [Test, MoqAutoData]
+        public async Task GetAllLocations_InvalidRequest_ReturnsLocations(
```
Comment (MarkFCain) [inline src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.Api.UnitTests/Controllers/ProviderLocationsControllerTests/GetProviderLocationTests.cs:37]: rename test, this is GetProviderLocation

**Review by MarkFCain — COMMENTED** [review]
Codebase looks fine, some queries about naming to address

File: src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.Api/Controllers/ProviderLocationsGetController.cs (hunk 1 above)
Comment (DevenKShah) [reply src/RoatpCourseManagement/SFA.DAS.RoatpCourseManagement.Api/Controllers/ProviderLocationsGetController.cs:1]: I agree the name does not sound right. Just avoid using GET in the name, for ex: ProviderCourseLocationsController

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**